daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Blinding.scala.orig

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
looks like you messed up a merge or something

Reply:
same thing in a few other places

Reply:
Fixed.